src/views/index.njk

```diff
@@ -1,7 +1,6 @@
 {% extends "layout-debug.njk" %}
 
 {% block content %}
-<div class="govuk-o-main-wrapper">
```

Comment:
Why was this removed? There is now no space between the top of the page and the heading on the index page.

Reply:
my bad, moved the other one into the logic but forgot to undelete this one.